modules/cert-manager-certificate-ingress.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-creating-certificate.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-certificate-ingress_{context}"]
+= Creating certificates for Ingress Controller
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have installed {cert-manager-operator} 1.13.0 or above.
+
+.Procedure
+
+. Create an issuer. For more information, see "Configuring an issuer" in "Additional Resources" section.
```

Comment:
Same comment about this maybe being more of a prereq than a first step, up to you.